### PR TITLE
Add basic arc diff commands.

### PR DESCRIPTION
**Pull request by dierksen**
- minor tweaks to `launch.json` for debugging.
- changed existing lint function name to replace actual use.

### CODE REVIEW

File: src/arc.ts [hunk 1]
```diff
@@ -49,6 +50,33 @@ export async function runArcDiff(fileName: string): Promise<ArcMessage[]> {
   return fileOutput;
 }
 
+export async function runArcDiff(flags?: string[]) {
+  const bin = config.pathToArc;
+  const command = [bin, "diff"];
+  const args = ["--config", "editor='code -w'"];
```
Comment (jdxcode) [inline src/arc.ts]: This won’t work if it’s running in remote ssh will it?

File: src/arc.ts [hunk 2]
```diff
@@ -36,7 +37,7 @@ interface ArcJSON {
   [filename: string]: ArcMessage[];
 }
 
-export async function runArcDiff(fileName: string): Promise<ArcMessage[]> {
```
Comment (jdxcode) [inline src/arc.ts]: Strange. I have no idea why I called it that

File: src/arc.ts (hunk 1 above)
Comment (dierksen) [reply src/arc.ts]: The remote ssh server extension appears to remap `code` to open on the remote client.

```
$ which code
/home/dierksen/.vscode-server/bin/86405ea23e3937316009fc27c9361deee66ffbf5/bin/code
```

Tested by connecting to devbox, and running `arc diff --config editor="code -w"`

Comment (jdxcode) [reply src/arc.ts]: Nice! Thanks for checking :+1:

**Pull request merged**